config/rbac/secret_editor_role.yaml

```diff
@@ -0,0 +1,31 @@
+# permissions for end users to edit secrets.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    app.kubernetes.io/name: clusterrole
+    app.kubernetes.io/instance: secret-editor-role
+    app.kubernetes.io/component: rbac
+    app.kubernetes.io/created-by: multi-cluster-traffic-controller
+    app.kubernetes.io/part-of: multi-cluster-traffic-controller
+    app.kubernetes.io/managed-by: kustomize
+  name: secret-editor-role
+rules:
+- apiGroups:
+  - kuadrant.io
```

Comment:
this seems wrong?

Reply:
yeah, this is left over from using kubebuilder to make the controller boilerplate, I'll remove it.